catalog/justfile

```diff
@@ -63,6 +63,10 @@ up *flags:
 up-deps *flags:
     env COMPOSE_PROFILES="catalog_dependencies" just ../up {{ flags }}
 
+recreate:
+    just ../down -v
```

Comment:
Should this have the following to prevent the catalog recreation from deleting other volumes (similar to the `up` recipe in this file)?
```suggestion
    env COMPOSE_PROFILES="catalog" just ../down -v
```

Reply:
Profiles are not used during `down`: https://github.com/docker/compose/issues/8139#issuecomment-1113185262

Reply:
Oh...well, that's unfortunate 😅